package/src/external/reanimated/moduleWrapper.ts

```diff
@@ -0,0 +1,33 @@
+import { useMemo } from "react";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+let Reanimated: any;
+
+try {
+  Reanimated = require("react-native-reanimated");
+} catch (e) {
+  // Ignore
+}
+
+export const HAS_REANIMATED = Reanimated !== undefined;
```

Comment:
this should be `export const HAS_REANIMATED = !!Reanimated;` (`Reanimated` will be false on RN Web with wepback).